spec/granite/query/assemblers/mysql_spec.cr

```diff
@@ -3,13 +3,19 @@ require "../spec_helper"
 {% if env("CURRENT_ADAPTER").id == "mysql" %}
```

Comment:
Add specs for `group_by`.  Include a test that uses multiple columns i.e. `Post.group_by([:name, :age])`